### PR TITLE
Web platform march 2022

### PR DESCRIPTION
**Pull request by rachelandrew**
The new to the web platform in March post. To do:

- [x] Link up Chromium beta post once it's available.

### CODE REVIEW

File: src/site/content/en/blog/web-platform-03-2022/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "New to the web platform in March"
+subhead: >
+  Discover some of the interesting features that landed in stable and beta web browsers during March 2022. 
+description: >
+  Discover some of the interesting features that landed in stable and beta web browsers during March 2022.
+date: 2022-03-31
+hero: image/kheDArv5csY6rvQUJDbWRscckLr1/PP0tuaoQwnakBvk1LFMT.jpg
```
[the remark this answers is not in the document]
Comment (malchata) [reply src/site/content/en/blog/web-platform-03-2022/index.md:9]: Per the images and markup guidelines, hero images should be 3200x960.

File: src/site/content/en/blog/web-platform-03-2022/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+```css
+h1 {
+  color: hwb(194 0% 0% / .5) /* #00c3ff with 50% opacity */
+}
+```
+
+{% BrowserCompat 'css.types.color.hwb' %}
+
+Firefox 99 includes the [pdfViewerEnabled](https://developer.mozilla.org/docs/Web/API/Navigator/pdfViewerEnabled) property of the Navigator interface. This property indicates if the browser supports inline display of PDF files.
```
Comment (malchata) [inline src/site/content/en/blog/web-platform-03-2022/index.md:66]:
```suggestion
Firefox 99 includes the [`pdfViewerEnabled`](https://developer.mozilla.org/docs/Web/API/Navigator/pdfViewerEnabled) property of the Navigator interface. This property indicates if the browser supports inline display of PDF files.
```

**Review by malchata — APPROVED** [review]
LGTM, just some minor suggestions that you can take or leave.

File: src/site/content/en/blog/web-platform-03-2022/index.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+h1 {
+  color: hwb(194 0% 0% / .5) /* #00c3ff with 50% opacity */
+}
+```
+
+{% BrowserCompat 'css.types.color.hwb' %}
+
+Firefox 99 includes the [`pdfViewerEnabled`](https://developer.mozilla.org/docs/Web/API/Navigator/pdfViewerEnabled) property of the Navigator interface. This property indicates if the browser supports inline display of PDF files.
+
+```
```
Comment (rachelandrew) [inline src/site/content/en/blog/web-platform-03-2022/index.md:68]:
```suggestion
```js
```

File: src/site/content/en/blog/web-platform-03-2022/index.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+Safari 15.4 also completed support for the [`:focus-visible`](https://developer.mozilla.org/docs/Web/CSS/:focus-visible) pseudo-class. The [implementation work](https://blogs.igalia.com/mrego/2021/06/07/focus-visible-in-webkit-may-2021/) on this was by Igalia.
+
+{% BrowserCompat 'css.selectors.focus-visible' %}
+
+## Beta browser releases
+
+Beta browser versions give you a preview of things that will be in the next stable version of the browser. It's a great time to test new features, or removals, that could impact your site before the world gets that release.
+
+New betas in March were Chrome 101, and [Firefox 99](https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/99).
```
Comment (rachelandrew) [inline src/site/content/en/blog/web-platform-03-2022/index.md:54]:
```suggestion
New betas in March were [Chrome 101](https://blog.chromium.org/2022/03/chrome-101-federated-credential.html), and [Firefox 99](https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/99). 
```

**Pull request merged**